docs/docs/tutorials/basics/index.mdx

```diff
@@ -0,0 +1,17 @@
+# Feldera Continuous Analytics Platform: the Basics
+
+In this tutorial we cover the basics of using Feldera, including:
+
+- Writing and testing SQL programs
+- Building continuous analytics pipelines
+- Managing pipelines using Feldera Web Console
+
+Along the way, we will introduce fundamental concepts behind Feldera:
+
+- Continuous analytics
+- Incremental query evaluation
+- Computation over data-in-motion
+
+> **_NOTE:_**  Throughout the tutorial we will encounter limitations of the
```

Comment:
I would just drop this. If someone really works through it they will encounter them we don't have to apologize upfront.

Reply:
Fair enough.  I wanted to apologize once, so I didn't need to apologize every time, but yeah, this isn't really necessary.